.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+name: lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  superlinter:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Full git history is needed to get a proper
+          # list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: Run super-linter
+        uses: super-linter/super-linter/slim@v7.1.0
+        env:
+          DEFAULT_BRANCH: main
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need to give it the token?

Reply:
The super-linter documentation says:
```
# To report GitHub Actions status checks
GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Not sure what exactly this means. I will check what happens without the token...

Reply:
Doesn't work without:

```
Failed to get [GITHUB_TOKEN]. Terminating because status reports were explicitly enabled, but GITHUB_TOKEN was not provided.
```

Reply:
What about:
```
    RUN_LOCAL: "true"
    DEFAULT_WORKSPACE: $CI_PROJECT_DIR
```

I always dislike giving some webservice our access Token, that is not needed for it to run.

Reply:
Nice. That seems to work!
With this there's no longer the "superlinter summary" on the action page on github, but that's ok.
![Screenshot from 2024-09-26 16-12-08](https://github.com/user-attachments/assets/7320bef2-7f08-43ca-8cd9-5095e76bae85)